spec/controllers/trips_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe TripsController, type: :controller do
+
+  describe "GET #show" do
```

Comment:
Let's slow down here a bit and focus on first of all trying to cover the unhappy cases for controllers. Not only for `read` actions but especially for the `write` ones: `update`, `create`, `destroy`. 

Also look at the descriptions for the specs. Usually one word there is I think incorrect there.

Edit: Do those tests pass?